src/Robo/Commands/Vm/VmCommand.php

```diff
@@ -118,6 +124,31 @@ protected function install() {
    * @command vm:config
    */
   public function config() {
+    $drupalvm_install_dir = $this->askDefault("Where would you like to install Drupal-VM files?", $this->vmConfigDir);
```

Comment:
I'm fine with allowing people to overwrite the location of the VM, but I'd rather not prompt them to do it. There should be a bit more ownership of the customization IMO.